ImmersionThemePlugin.inc.php

```diff
@@ -183,8 +183,7 @@ public function addIssueTemplateData($hookname, $args)
         $immersionSectionColors = $issue->getData('immersionSectionColor');
         if (empty($immersionSectionColors)) return false; // Section background colors aren't set
 
-        $sectionDao = DAORegistry::getDAO('SectionDAO'); /** @var SectionDAO $sectionDao */
-        $sections = $sectionDao->getByIssueId($issue->getId()); /** @var Section[] $sections */
+        $sections = Repo::section()->getByIssueId($issue->getId()); /** @var \Illuminate\Support\LazyCollection $sections */
```

Comment:
No need to fix this. Return type is already specified for these new methods, thus phpdoc isn't required here or we can specify the type of individual items (as you already done below), although the latter is supported differently in IDEs